custom/logistics/tasks.py

```diff
@@ -68,6 +70,7 @@ def stock_data_task(domain, endpoint, apis, test_facilities=None):
         )
         limit = 100
         offset = 0
+        # todo: kkrampa why is this here?
```

Comment:
Basically, It's caused by way that we are migrating data. First of all we are migrating all product stocks for all locations, then all stock transactions for all locations etc. Yeah, I know that probably It should be  flipped (get data location by location) but probably it doesn't speed up migration.